support/cas-server-support-inwebo-mfa/src/main/java/org/apereo/cas/support/inwebo/service/response/AbstractResponse.java

```diff
@@ -0,0 +1,21 @@
+package org.apereo.cas.support.inwebo.service.response;
+
+import lombok.Getter;
+import lombok.Setter;
+
+/**
+ * The abstract JSON response.
+ *
+ * @author Jerome LELEU
+ * @since 6.4.0
+ */
+@Getter
+@Setter
+public abstract class AbstractResponse {
```

Comment:
Please make sure all model classes in the response or soap packages are prefixed with Inwebo.

Reply:
Sure. I'd prefer keep the original names for the generated SOAP classes. They are already in a `soap` package.